docs-source/content/faq/oci-fss-pv.md

```diff
@@ -0,0 +1,96 @@
+---
+title: "Using OCI File Storage (FSS) for Persistent Volumes"
+date: 2020-02-12T12:12:12-05:00
+draft: false
```

Comment:
This should have a weight in order to choose where it ends up in the FAQ contents list.  See other FAQs for their weights.

Reply:
I didn't and don't see a weight on any FAQ file?

Reply:
You're right. The FAQs do not have weights, not sure why other than perhaps we have not been particular about their order.